test/unit/top-app-bar/Icon.test.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
```

Comment:
can you add a test where the child is an inline SVG?

Reply:
yes. done :)

Reply:
Thanks!